TwL updates PR (#8744)
DataDog/documentation: content/en/tracing/trace_retention_and_ingestion/_index.md

| Spans Ingested            | Number of spans ingested by Datadog over the selected time period.        |

### Change the Default Ingestion Rate

{{< img src="tracing/trace_indexing_and_ingestion/ChangeIngestRate3.gif" style="width:100%;" alt="Change the Data Ingestion Rate" >}}

To specify that a specific percentage of a service's traffic should be sent, add a generated code snippet to your tracer configuration for that service.

1. Select the service you want to change the ingested span percent for.
2. Choose the service language.
3. Choose the desired ingestion percentage.
4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy.
5. Confirm on the Data Ingestion page that your new percentage has been applied.

#### (Recommended) Setting the global ingestion rate to 100%

In order to ingest 100% of your traces in Datadog for all services for live search and analytics as well as to have the most control with retention filters, Datadog recommends configuring all services to send 100% of traces by default.

> This can be done with one configuration on every service instrumented with a Datadog tracing library by setting the environment variable below in the tracer configuration:

Reviewer: ```suggestion
To configure for 100% ingestion on every service instrumented with a Datadog tracing library, set the following environment variable in the tracer configuration:
```

```
DD_TRACE_SAMPLE_RATE=1.0
```

### Traces dropped before ingestion

> If you have not set the environment variable configuration `DD_TRACE_SAMPLE_RATE=1.0` for Tracing without Limits and
> - your applications generate above 50 traces per second
> - your applications send intermittent traffic loads
> - your applications traces are large in size or otherwise have complicated trace payloads
> 
> This may make them not always send 100% of your traces to Datadog.  In this case, some traces will be dropped en route to Datadog.  These traces will be dropped in the Datadog Agent, *after* stats are computed, so that metrics calculated will be based on 100% of your traces.

Reviewer: ```suggestion
You won't get 100% trace ingestion if you have not set the environment variable configuration `DD_TRACE_SAMPLE_RATE=1.0` for Tracing without Limits, and:
- your applications generate above 50 traces per second;
- your applications send intermittent traffic loads; or
- your applications traces are large in size or otherwise have complicated trace payloads.

In this case, some traces will be dropped by the Datadog Agent *after* stats are computed, so that metrics calculated will be based on 100% of your traces.
```

> In case you are seeing ingestion rates below 100% within Datadog and would like to send all your traces, our recommendation is to enable Tracing without Limits. If you have any questions, feel free to reach out to our [support team][5].

Reviewer: ```suggestion
If you are seeing ingestion rates below 100% within Datadog and would like to send all your traces, enable Tracing without Limits by setting the environment variable as described above. If you have questions, contact our [support team][5].
```

{{< img src="tracing/trace_indexing_and_ingestion/VisualIndicator.png" style="width:100%;" alt="Root services not sending 100% of traces" >}}

## App Analytics to Tracing Without Limits

> Prior to October 20, 2020, Datadog offered App Analytics to index spans for performing analytics. While this is no longer the recommended setup configuration and is not needed to use [Trace Search and Analytics][6], the legacy instructions are available within the [App Analytics][7] setup page.

Reviewer: ```suggestion
Before October 20, 2020, Datadog offered App Analytics to index spans for performing analytics. While this is no longer the recommended setup configuration and is not needed to use [Trace Search and Analytics][6], the legacy instructions are available within the [App Analytics][7] setup page.
```
